var/spack/repos/builtin/packages/py-pywavelets/package.py

```diff
@@ -35,7 +35,7 @@ class PyPywavelets(PythonPackage):
 
     version('0.5.2', 'aedda732f064cf9395f03d37f1003d1a')
 
-    import_modules = ['pywt']
+    import_modules = ['pywt', 'pywt._extensions', 'pywt.data']
```

Comment:
I would leave out `pywt._extensions`. Things that start with an underscore are usually "hidden" things used internally and aren't meant to be user-facing.